spec/features/contestants/index_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Contestand Index', type: :feature do
+  describe 'as a user' do
+    it 'when I visit contestant index page, I should see' do
+      recycled_material_challenge = Challenge.create(theme: 'Recycled Material', project_budget: 1000)
+      news_chic = recycled_material_challenge.projects.create(name: 'News Chic', material: 'Newspaper')
+      jay = Contestant.create(name: 'Jay McCarroll', age: 40, hometown: 'LA', years_of_experience: 13)
+      ContestantProject.create(contestant_id: jay.id, project_id: news_chic.id)
```

Comment:
side note, you can also do `ContestantProject.create(contestant: jay, project: news_chic)` and rails will know to match the id up. Either way works!